src/export.cpp

```diff
@@ -1096,18 +1095,22 @@ void SolveSpaceUI::ExportAsPngTo(const std::string &filename) {
 #if !defined(WIN32)
     GlOffscreen offscreen;
     offscreen.Render((int)SS.GW.width, (int)SS.GW.height, [&] {
-        SS.GW.Paint();
     });
+#else
+    SS.GW.Paint();
 #endif
-    SS.showToolbar = prevShowToolbar;
-
     // Somewhat hacky way to invoke glReadPixels without dragging in all OpenGL headers.
     OpenGl1Renderer canvas = {};
     canvas.camera = SS.GW.GetCamera();
+    SS.showToolbar = false;
```

Comment:
All the other modifications should not be necessary once the `#else` branch is added.